modules/gdscript/gdscript.h

```diff
@@ -33,10 +33,11 @@
 
 #include "core/debugger/engine_debugger.h"
 #include "core/debugger/script_debugger.h"
+#include "core/doc_data.h"
 #include "core/io/resource_loader.h"
 #include "core/io/resource_saver.h"
 #include "core/object/script_language.h"
-#include "editor/doc_data.h"
+#include "core/doc_data.h"
```

Comment:
You include this twice.

Reply:
done the merge conflicts and style checks, I fix the  encapsulation issue ASAP